app/app.go

```diff
@@ -977,6 +979,8 @@ func New(
 
 	app.RegisterDeliverTxHook(app.AddCosmosEventsToEVMReceiptIfApplicable)
 
+	app.txPrioritizer = NewSeiTxPrioritizer(&app.EvmKeeper, &app.UpgradeKeeper, &app.ParamsKeeper).GetTxPriority
```

Comment:
Nit: fix indent?

Reply:
Sorry, in what sense? would you like to see a new line or?